### PR TITLE
Update test-infra repo admins and maintainers

### PR DESCRIPTION
**Pull request by dims**
active folks from: https://github.com/kubernetes/test-infra/blob/master/OWNERS_ALIASES#L125-L133

### CODE REVIEW

File: config/kubernetes/sig-testing/teams.yaml
Comment (michelle192837) [inline config/kubernetes/sig-testing/teams.yaml:67]: I'd like to leave the folks who are still oncall (chases2, listx, mpherman2)

But aside from that, LGTM!

**Review by michelle192837 — APPROVED** [review]
/hold

For others to review

File: config/kubernetes/sig-testing/teams.yaml
Comment (dims) [reply config/kubernetes/sig-testing/teams.yaml:67]: @michelle192837 then shouldn't we also have the `sig-k8s-infra-oncall` as well?

https://github.com/kubernetes/test-infra/blob/master/OWNERS_ALIASES#L16C3-L22

Comment (michelle192837) [reply config/kubernetes/sig-testing/teams.yaml:67]: That's fine by me 👍

**Comment by dims** [comment]
it's been a week, let's merge and iterate if needed.

/hold cancel

**Comment by dims** [comment]
will file a follow up PR with the rest of the crew once this merges

**Pull request merged**
